src/q8/sigproc-ELEC2900/exam/2019/Juin/All/sigproc-ELEC2900-exam-2019-Juin-All.tex

```diff
@@ -0,0 +1,107 @@
+\documentclass[en]{../../../../../../eplexam}
+
+\usepackage{../../../../../../eplcode}
+
+\hypertitle{Signal processing}{8}{ELEC}{2900}{2019}{Juin}{All}
+{Olivier Leblanc \and Julien Verecken}
+{Luc Vandendorpe, Benoît Macq and Laurent Jacques}
+
+Exam during 4h with 6 sheets of questions: 2 questions of LV, 3 questions of LJ et one bonus question about the Python sessions.
+
+\section{Questions of Luc Vandendorpe}
+
+\subsection{LV1}
+
+We are given a signal $x_5[n]$ sampled at 5Ms/s, grouped into blocks of length $N$. We would like to produce the signal $x_6[n]$ at 6Ms/s and use the DFT operator in this aim.
```

Comment:
Is the "x Ms/s" notation used in class or did you come up with it on your own? If not, I'd advise you set up a `\mss` unit with `siunitx` which shows as "xM samples/s". If I'm being overly pedantic and this is standard notation, disregard this comment.

Reply:
Even though I approve your point, I guess that writing "sampled at 5 M samples/s" nearly everywhere is quite verbose, and as it takes more characters, I wouldn't be surprised if they used "Ms/s".

Reply:
Hence why I had the last sentence in my comment :).

Reply:
Do we keep it like this?

Reply:
My opinion is to keep it like this.